rest-api/src/test/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/AdminWebAuthorizationControllerTest.java

```diff
@@ -94,7 +94,7 @@ void getAdminWebAuthorization() throws Exception {
     private AdminWebAuthorization callApi() throws Exception {
         final MvcResult
             mvcResult = mockMvc.perform(get("/api/idam/adminweb/authorization")).andExpect(status().isOk()).andReturn();
-        assertThat(mvcResult.getResponse().getContentType(), is("application/json;charset=UTF-8"));
+        assertThat(mvcResult.getResponse().getContentType(), is("application/json"));
```

Comment:
Deprecate MediaType.APPLICATION_JSON_UTF8 in favor of APPLICATION_JSON from Spring 5.2
https://github.com/spring-projects/spring-framework/issues/22788